chat-client/src/client/chat.ts

```diff
@@ -51,12 +57,10 @@ const DEFAULT_TAB_DATA = {
 
 type ChatClientConfig = Pick<MynahUIDataModel, 'quickActionCommands'>
 
-export const createChat = (
-    clientApi: { postMessage: (msg: UiMessage | ServerMessage) => void },
-    config?: ChatClientConfig
```

Comment:
Won't we need to support passing configuration from client at initialisation time? If yes, we should keep `config` param in function signature.